jsapp/js/components/common/icon.tsx

```diff
@@ -9,18 +9,20 @@ import './icon.scss';
 export type IconSize = 'l' | 'm' | 's' | 'xl' | 'xs' | 'xxs';
 
 const DefaultSize = 's';
+const DefaultColor = 'inherit';
 
 interface IconProps {
   name: IconName;
   size?: IconSize;
   classNames?: string[];
+  color?: string;
```

Comment:
Let's make this not any `string`, but a list of limited colors available (from our colors list). Similarly to `export type ButtonColor = 'blue' | 'light-blue' | 'red' | 'storm';`, but instead of defining all that make sense, please start with the ones you use :)

Reply:
I was curious if this was the way we wanted to go about it because there wasn't a good way to just assign variables. I will update